README.md

```diff
@@ -1 +1,58 @@
-# Bookstore-react
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> This is a bookstore project built with react and redux.
+
+## Built With
+
+- JavaScript
+- React
+- Redux
+- Node.js
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://reverent-joliot-93b9ef.netlify.app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1- clone the repo to your local pc with this link (<https://github.com/AkintoyeOlamide/Bookstore-react>).
+2- cd into the directory using `cd BookStore` command.
```

Comment:
- [ ] [REQUIRED] Please, make sure that info in your readme is correct. Here for examlpe, users should use commande `cd Bookstore-react`. Keep in mind that your readme is the first thing users check. You can also use tools like grammarly to help you with your grammar :smile: